src/core.js

```diff
@@ -789,9 +789,11 @@ goog.require('olcs.core.OlLayerPrimitive');
       var position = olcs.core.ol4326CoordinateToCesiumCartesian(center);
       var color;
       var opacity = imageStyle.getOpacity();
-      if (goog.isDef(opacity)) {
-        color = new Cesium.Color(1.0, 1.0, 1.0, opacity);
+      if (!goog.isDef(opacity)) {
+        opacity = 1;
       }
+      color = new Cesium.Color(1.0, 1.0, 1.0,
+          opacity * billboards.olLayerOpacity);
```

Comment:
The layer opacity may be different from the default; that is why it is necessary to read the actual value.